src/reyna.Facts/GivenAForwardService.cs

```diff
@@ -2,21 +2,38 @@
 {
     using System;
     using System.IO;
+    using System.Net;
     using System.Reflection;
     using System.Threading;
     using Moq;
+    using OpenNETCF.Net.NetworkInformation;
     using Reyna.Interfaces;
     using Xunit;
 
     public class GivenAForwardService
     {
         public GivenAForwardService()
         {
+            NetworkInterface.NetworkInterfaces = new INetworkInterface[0];
+            this.Preferences = new Preferences();
```

Comment:
The issue regarding these that, I need to test the main logic without breaking it into smaller,, protected function.
So I used thread sleep to simulate message added to sore and try to send it once added.
Can lead to false alert test? but still better to test full scenario.